### PR TITLE
"Highlights" section on homepage

### PR DESCRIPTION
**Pull request by gordoncc**
3-card module that will link to a the roadmap, hoon school, and a forth-coming blog post.

(Don't deploy until blog post has been published so we can link to it there.)

### CODE REVIEW

**Comment by thomas-kroes** [comment]
Looks great. I think a bit more activating text would benefit conversion for HSL and the other CTAs

File: content/highlights/highlight-2.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
++++
+title = "The Roadmap"
+image = "https://storage.googleapis.com/media.urbit.org/site/featured/roadmap-icon.jpg"
+url="https://roadmap.urbit.org/"
+description = "Explore the technical plan for making Urbit ready for mass adoption."
```
Comment (jalehman) [inline content/highlights/highlight-2.md:5]:
```suggestion
description = "The plan for making Urbit ready for mass adoption."
```

File: content/highlights/highlight-3.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
++++
+title = "Hoon School Live 23'"
```
Comment (jalehman) [inline content/highlights/highlight-3.md:2]:
```suggestion
title = "Hoon School Live March 23'"
```

File: content/highlights/highlight-3.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+image = "https://storage.googleapis.com/media.urbit.org/site/featured/hoon-school-23.png"
+url="https://developers.urbit.org/courses/hsl"
+description = "Sign up now for Hoon School Live and start your Hoon journey in March"
```
Comment (jalehman) [inline content/highlights/highlight-3.md:5]:
```suggestion
description = "Sign up now to learn the basics of Urbit development"
```

**Review by jalehman — APPROVED** [review]
Copy suggestions

**Pull request merged**